modules/rbac/provisioner.tf

```diff
@@ -0,0 +1,21 @@
+#resource "null_resource" "enable_azure_rbac" {
```

Comment:
remove unnecessary code if no need, 
If you need this code add TODO: description